internal/circle/client.go

```diff
@@ -282,7 +315,7 @@ func (client *Client) handleError(ctx context.Context, resp *http.Response) erro
 		return fmt.Errorf("parsing API error: %w", err)
 	}
 
-	return fmt.Errorf("Circle API error: %w", apiError) //nolint:golint,unused
+	return fmt.Errorf("circle API error: %w", apiError) //nolint:golint,unused
```

Comment:
Do we need the comment `//nolint:golint,unused`?

Reply:
I didn't add it, but I'll remove it :)

Reply:
addressed in a712a9b61f6dafc8344674f3adc55c3525d11617